src/models/Settings.js

```diff
@@ -79,6 +79,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, config) {
       'features.preventBrowserFromSavingOktaPassword': ['boolean', true, true],
       'features.deviceFingerprinting': ['boolean', false, false],
       'features.hideSignOutLinkInMFA' : ['boolean', false, false],
+      'features.hideBackToSignInForReset' : ['boolean', false, false],
```

Comment:
should this be renamed to `hideBackToSignInForResetAndRecovery` ? since I see this is also used in some recovery flows ?